bundles/core/org.eclipse.smarthome.core.thing.xml.test/src/test/resources/test-bundle-pool/SystemChannels.bundle/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Synthetic Test Bundle
+Bundle-SymbolicName: SystemChannels.bundle;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.7.0.qualifier
```

Comment:
we can do so - I've copied this from the other synthetic bundles in this .test bundle so they will also need to be edited